shell/app/common/components/pagination/index.tsx

```diff
@@ -72,15 +72,13 @@ const Pagination = (pagination: IPaginationProps) => {
 
   const pageSizeMenu = (
     <Menu>
-      {(pageSizeOptions || [])
-        .filter((item) => item)
-        .map((item: string | number) => {
-          return (
-            <Menu.Item key={item} onClick={() => onChange?.(1, +item)}>
-              <span className="fake-link mr-1">{i18n.t('{size} items / page', { size: item })}</span>
-            </Menu.Item>
-          );
-        })}
+      {PAGINATION.pageSizeOptions.map((item: string | number) => {
```

Comment:
in line `103`， show this dropdown still depend on `pageSizeOptions?.length` .  So I dont understand your change logic

Reply:
I forgot to remove the logic in line 103, and then the pageSizeOptions is fixed.